docs/src/components/ExternalExampleLayout.tsx

```diff
@@ -1,3 +1,4 @@
+import { Provider } from '@stardust-ui/react'
```

Comment:
Can we move imports from our internal project after the imports form external dependencies. It is much clearer when you look them to figure out what is coming form where. This is my personal opinion, let me know what you think.

Reply:
I want to introduce ESLint/TSLint and align all imports in future
I placed  `@stardust-ui/react` at first because namespaced imports have more priority